docs/faq/network/node-setup.md

```diff
@@ -0,0 +1,106 @@
+
+- **[Lighthouse](https://github.com/sigp/lighthouse)** is an Ethereum consensus client written in Rust and developed by Sigma Prime. From a security perspective, Lighthouse leverages Rust's safety features and undergoes regular security audits to protect against potential vulnerabilities. Regarding efficiency, Lighthouse is designed to perform well even on low-spec hardware, making it accessible to a wide range of users with different skill levels.
+
+### Should I run an archive or a full node?
+
+A full node downloads the entire blockchain and validates all blocks and transactions against the network's consensus rules. It stores the **current state of the network, including account balances, contracts, storage, and other informatio**n. However, it does not keep all historical states. If you need to check the balance of an account at a specific block height in the past, a full node cannot provide this information directly.
+
+An archive node is a particular type of full node. It also downloads the entire blockchain and validates all blocks and transactions like a full node. In addition to the current state of the network, it **also stores all historical states since the genesis block**. Keeping the entire historical state makes an archive node much more storage extensive than a full node, but it allows you to query any historical state directly on the node.
+
+If you are running a validator, it's the default and recommended option to go with a full node. Archive nodes should only be run by professional setups or data centers with lots of storage.
+
+### Can the LUKSO CLI run as an archive node?
+
+By default, the LUKSO CLI is running as a full node. However, you can modify the configuration files and pass any flags to the execution clients by adding the `--geth-[parameter]` or `--erigon-[parameter]` flags to the start command.
+
+### How much storage does my node need?
+
+The needed storage can be broken down into yearly growth based on other EVM networks and depends on whether you are running an archive node or a full node. We can make a rough estimate by analyzing QuickNode, Ledgerwatch, and YCharts. By default, the LUKSO CLI should store about `120 GB` per year using Geth and about `60 GB` per year using Erigon. If you pass down archive node flags, the storage will increase to about `1.8 TB` per year on Geth and about `320 GB` per year using Erigon.
+
+### Will there be a GUI for running a node?
+
+The [LUKSO Launchpad](https://deposit.mainnet.lukso.network/) will streamline the process of generating deposit data and transferring validator funds. However, the node setup and maintenance will require using the terminal for both [LUKSO CLI](https://github.com/lukso-network/tools-lukso-cli) and [Docker](https://github.com/lukso-network/network-docker-containers).
+
+### How can I check if my node is working fine?
```

Comment:
using the consensus explorer also to find if your validators are up and running

Reply:
This is the node setup. But agree, it might be a good addition for the validator section